src/Models/product/Transformers/ProductTransformer.ts

```diff
@@ -29,11 +29,11 @@ export default class ProductTransformer extends BaseTransformer {
             metaDescription: product.metaDescription,
             shortDescription: product.shortDescription,
             // has many
-            variants: this.includeProductVariant(product.variants || []),
+            variants: this.includeProductVariant(product),
         });
     }
 
-    includeProductVariant(product: ProductInterface[]): ProductVariantInterface[] {
+    includeProductVariant(product: ProductInterface): ProductVariantInterface[] {
```

Comment:
I think this method name should be plural
```suggestion
    includeProductVariants(product: ProductInterface): ProductVariantInterface[] {
```